### DISCUSSION TITLE
Welcome to APMIS-Project Discussions!

### DISCUSSION THREAD

**Discussion by danieltomabba** [xlg8/APMIS-Project#649] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sundayAdu** [comment]
great!

**Reply by naveeddil** [reply]
Unable to change "Lot Cluster Number" variable for LQAS lots although. i.e lot Cluster 5 is not available in Achin (East) but still it is giving an error

![LQAScluster error](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/94aff44c-2431-4229-9ad1-d70002fcddd2)

**Reply by naveeddil** [reply]
Just now tried another lot and found that Open any cluster in LQAS and **click Save button without any** **changes and same message appears**

**Reply by danieltomabba** [reply]
Hi @naveeddil, @sundayAdu took note of this and they are working on this. He will get back to you on this. Thanks

**Reply by sundayAdu** [reply]
We are currently working on this and will revert soonest.

**Reply by naveeddil** [reply]
@sundayAdu !  Looking forward as we have alot of changes to do for NID April before export and utilize the data.

**Reply by naveeddil** [reply]
Any update on its resolution ?

**Reply by sundayAdu** [reply]
Apologies for slow response @naveeddil on this issue, we are looking at the best window to update the system with the patch to fix the issue identified. However, we have been asked in the past not to update/patch the system while in active campaign. @danieltomabba could you give advise if this should be patched and when will be best time to carry out patch on the Liver server? Thank you.

**Comment by sundayAdu** [comment]
@naveeddil We apologize for the delay. Both the live server and the mobile app have been patched. However, you will need to download the APK and reinstall it to receive the update. We plan to upgrade the APK to a higher version later to enable automatic updates.

Thank you, and please reach out if you encounter any further issues.

**Reply by naveeddil** [reply]
after installing new update from the server, Now unable to login back to the application. @RaziaMahmodi to verify

**Reply by naveeddil** [reply]
![Error](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/70272bb1-0bce-4317-ab78-d07471565ab3)

Web also have errors

**Reply by sundayAdu** [reply]
check this out.

**Comment by sundayAdu** [comment]
@naveeddil This has now been resolved
